runtime/oti/j9vm.hdf

```diff
@@ -1218,7 +1218,7 @@ typedef UDATA (* lookupNativeAddressCallback)(struct J9VMThread *currentThread,
 		<data type="struct J9VMThread *" name="currentThread" description="current thread" />
 		<data type="I_64" name="millis" description="milliseconds to sleep for" />
 		<data type="IDATA" name="nanos" description="nanoseconds to sleep for" />
-		<data type="IDATA" name="startNanos" description="current time in nanos when sleep began" />
+		<data type="I_64" name="startNanos" description="current time in nanos when sleep began" />
```

Comment:
Perhaps `nanos` should be `I_64` as well?

Reply:
The Java API is defined as `Thread::sleep(long millis, int nanos)`

Reply:
Then `nanos` should be `I_32_ instead of `IDATA`.

Reply:
I have no problem making that change. Just want to hear from @gacholio if there was a reason why we went with `IDATA` instead of `I_32`